articles/azure-app-configuration/push-kv-devops-pipeline.md

```diff
@@ -101,3 +102,7 @@ Create multiple instances of the Azure App Configuration Push task within the sa
 **Why am I receiving a 409 error when attempting to push key-values to my configuration store?**
 
 A 409 Conflict error message will occur if the task tries to remove or overwrite a key-value that is locked in the App Configuration store.
+
+**Why am I getting the "crypto.default.randomFillSync is not a function" error when attempting to push key values?**
+
+ The error is caused by the nodejs version that is currently being used by the agent. Update the agent or the agent's nodejs version to 6.13.0 or above.
```

Comment:
Suggested change:
The error occurs when the Azure Pipeline agent does not have the minimum required version of Node installed. Update the agent to the [latest version](/azure/devops/pipelines/agents/agents?view=azure-devops#how-do-i-make-sure-i-have-the-latest-v2-agent-version) or install Node version 6.13.0 or above in the step before running this task.